tests/resources/test_client.py

```diff
@@ -34,7 +34,10 @@ def test_init_with_access_keys(self):
             access_token_secret="bar",
         )
 
-        with self.assertRaisesRegex(errors.APIError, "404 NotFound"):
+        with self.assertRaisesRegex(
```

Comment:
Why did this change?

Reply:
It changed to become what it should have been in the first place actually haha, thanks for artichoke!
```
AssertionError: "404 NotFound" does not match "401 InvalidAuthentication: {'message': 'Invalid artichoke authentication', 'type': 'InvalidAuthentication'}"
```